include/wx/osx/bitmap.h

```diff
@@ -204,6 +204,9 @@ class WXDLLIMPEXP_CORE wxBitmap: public wxBitmapBase
     // returns an autoreleased version of the image
     WX_NSImage GetNSImage() const
         { return GetImage(); }
+
+    // Convert from wxCursor
+    wxBitmap(const wxCursor &cursor);
```

Comment:
Ctors are supposed to be all grouped together in the beginning of the class declaration.